editor/script_create_dialog.h

```diff
@@ -40,6 +40,8 @@
 #include "scene/gui/option_button.h"
 #include "scene/gui/panel_container.h"
 
+class CreateDialog;
```

Comment:
Forward declaration was needed because `#include "editor/create_dialog.h"` in the .h caused problems so I moved it to the .cpp